content/en/features/ocf-cloud-api.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 'OCF Cloud API for Cloud Services'
+description: 'How to validate conformance to OCF Cloud API for Cloud Services?'
+date: '2021-11-10'
+lastmod: '2021-11-11'
+categories: [features]
+keywords: ['OCF conformance', 'tutorial']
+menu:
+  docs:
+    parent: features
+    weight: 10
+toc: true
+---
+
+The OCF Cloud API for Cloud Services is a set of well-defined APIs used for device information retrieval, update and event subscription between two OCF Cloud instances. The [full specification](https://openconnectivity.org/specs/OCF_Cloud_API_For_Cloud_Services_Specification_v2.2.4.pdf) is available at The Open Connectivity Foundation (OCF) website.
+
+plgd.dev services `cloud2cloud-gateway` and `cloud2cloud-connector` provide an implementation of the APIs. To validate conformance OCF uses a Windows-based application called Conformance Test Tool (CTT). To succeed with the validation you must have a correctly set up environment and configuration file for CTT. The following sections will explain all the necessary steps.
+
+## Validating Target Cloud conformance with Conformance Test Tool
+
+The environment used in this tutorial consists of the following parts:
+
+* Windows 10 host machine with Ethernet connection
+* Ubuntu 20.04.3 LTS running in Oracle VM VirtualBox 6.1
+
+### Setup the Windows host machine
+
+* Install CTT on the host machine. The default installation should be located in `C:\Program Files (x86)\OCF Conformance Test Tool`.
+* Install [ngrok](https://ngrok.com/) and set it up as is described in the CTT Users Guide located in `C:\Program Files (x86)\OCF Conformance Test Tool\Docs\Users Guide.mhtml`. Follow the section named **C2C Accessibility**.
+* Examine the Ubuntu machine VirtualBox, go to Settings/Network pane. Make sure that at least one adapter is enabled and the "Attached to" setting is set to `Bridged Adapter` value.
+
+### Run OCF Cloud Bundle in Ubuntu machine
+
+The Target Cloud part of the API is implemented by the `cloud2cloud-gateway`, which is included by the plgd.dev [OCF Cloud Bundle implementation](https://github.com/plgd-dev/hub/tree/main/bundle). However, since the services must be available also on the Windows host machine, several environmental variables must be defined.
+
+Run the following commands in your `$HOME` folder in terminal:
+
+```bash
+docker run -it --rm \
+  -e FQDN=192.168.1.44 \
+  -e OAUTH_AUDIENCE=https://try.plgd.cloud \
+  -e OAUTH_ENDPOINT=auth.plgd.cloud \
+  -e OAUTH_CLIENT_ID=cYN3p6lwNcNlOvvUhz55KvDZLQbJeDr5 \
+  -e OAUTH_CLIENT_SECRET=qrccMDQjijrZDztK9-P151HgkDY1QqTh7WOhI3OT_0I0Mi53b-taxMC55iL4XYwa \
```

Comment:
@ondrejtomcik apparently, I should've used some testing clientId and clientSecret, but since I've published them here these are now the testing values. According @jkralik we should create a new clientId and secret and switch plgd.cloud to it. Anything else needs to be done?